### PR TITLE
Bianco ch6 response

### PR DESCRIPTION
**Pull request by fedhere**
Changes on chapter 6.3 probably dont need review but I also added a figure to chapter 6.2 : similarity matrx following the request to clarify the existing figures. Please let me know if its ok!

### CODE REVIEW

File: whitepaper/Transients/sn.tex [hunk 1]
```diff
@@ -107,43 +108,50 @@ \section{Supernovae as Transients}
...
+SNe to enable us to distinguish
 populations with different contribution from DD and SD progenitors.
-To do this we rely on simulations of the observables of the population
-for different sample sizes, and on the \texttt{transientAsciiMetric}
-to determine the detectability of interacting, and non-interacting
+We rely on simulations of the observables of the population for
+different sample sizes, and on the \texttt{transientAsciiMetric} to
+determine the detectability of interacting vs non-interacting
 SNe. We are developing a metric (\texttt{colorGapMetric}) to assess
 the gap between of detections in 2 filters. In the meantime we rely on
```
Comment (svalenti) [inline whitepaper/Transients/sn.tex]: typo: Remove "of"

File: whitepaper/Transients/sn.tex [hunk 2]
```diff
@@ -266,103 +286,104 @@ \subsection{Distinguishing progenitor scenarios}
...
+
+ \item[Q5:] {\it Does the science case place any constraints on the
+ fraction of observing time allocated to each band?}
+
+ \item[A5:] Yes, since it relies on obtaining observations in at least 2 filters.
+
+ \item[Q6:] {\it Does the science case place any constraints on the
+ cadence for deep drilling fields?}
+
+ \item[A6:] Yes, although the present simulation has not included deep drilling fields.
```
Comment (svalenti) [inline whitepaper/Transients/sn.tex]: is that true, I think 5 drilling fields are there

File: whitepaper/Transients/sn.tex [hunk 3]
```diff
@@ -266,103 +286,104 @@ \subsection{Distinguishing progenitor scenarios}
...
+
+ \item[Q7:] {\it Assuming two visits per night, would the science case
+ benefit if they are obtained in the same band or not?}
+
+ \item[A7:] No.
```
Comment (svalenti) [inline whitepaper/Transients/sn.tex]: I think we should write something like: "We have shown that the intra-night gap of ~30 minutes is too short to identify young transients. If the  intra-night gap will remain ~30 minutes, we would benefit from having the second visit in a different filter."

Comment (fedhere) [inline whitepaper/Transients/sn.tex]: this is a reply to the section 6.3 - for section 6.3 No is the right answer, but I added the last sentence

File: whitepaper/Transients/sn.tex [hunk 4]
```diff
@@ -359,12 +360,12 @@ \subsection{Distinguishing progenitor scenarios}
...
  cadence for deep drilling fields?}
 
- \item[A6:] Yes, although the present simulation has not included deep drilling fields.
+ \item[A6:] Yes, although the results have not been analyzed separately for WFD and DD fields.
 
  \item[Q7:] {\it Assuming two visits per night, would the science case
  benefit if they are obtained in the same band or not?}
 
- \item[A7:] No.
+ \item[A7:] No. Although  we would benefit greatly from 2 visits in the same filters, and one visit in a different filter to constrain simultaneously shape and color."
```
Comment (svalenti) [inline whitepaper/Transients/sn.tex]: remove "

**Comment by fedhere** [comment]
I can split the layout but those are not white spaces: for each transient pair there is a range of values depending on the  time   passed since explosion. There  is a sentence in the caption to that extent. I will try anc clear it up but if you have suggestions please go ahead!

**Comment by ebellm** [comment]
Okay, the caption helps--it may still be a bit complex for easy grokking, but let's get it in the whitepaper for now.

**Comment by drphilmarshall** [comment]
Thanks very much, all! :-)

**Pull request merged**